src/components/dashjs-api-control/dashjs-api-control.tsx

```diff
@@ -144,6 +162,78 @@ export class DashjsApiControl {
     toast.present();
   }
 
+  updateInformation(action : number, event : any) {
+    if(action === 0) {
+      console.log("saving: " + event.detail.checked);
+      saveStringLocally('api_autostart', event.detail.checked);
+    } else if(action === 1) {
+      if (event.detail.value == '') resetMediaURL();
+      else  setMediaURL(event.detail.value);
```

Comment:
`{` missing?

Reply:
{ } are not required if there is only one action for the ifs and elses, but I can add that if you don't like that style